cape/cape_main.py

```diff
@@ -2178,6 +2180,19 @@ def _extract_commands(self) -> None:
                 }
             )
 
+    def _extract_buffers(self) -> None:
```

Comment:
if you are writing a file to the disk during an execution, you have to include that file in the `_cleanup_leftovers()` method to ensure that that file does not exist upon the next execution